### PR TITLE
How to cite page

### PR DESCRIPTION
**Pull request by diegoonolasco**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -89,6 +89,13 @@ enableEmoji = true
         identifier = "collaborative-projects"
         weight = 3
 
+      [[menu.main]]
+        name = "How to cite"
+        url = "/science/how-to-cite/"
+        parent = "science"
+        identifier = "hot-to-cite"
```
Comment (j-wags) [inline config.toml]:
```suggestion
        identifier = "how-to-cite"
```

File: content/science/how-to-cite/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: How to Cite
+color: red
+class: color-footer
+---
+
+When using Open Force Field parameters or software in your research, please cite the appropriate works from below. If multiple citations are listed for an item, we recommend citing all of the suggested references.
+
+### Force fields
+
+* Parsley ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Zenodo DOIs](https://github.com/openforcefield/openff-forcefields#open-force-fields))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md]:
```suggestion
* Parsley ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://github.com/openforcefield/openff-forcefields#open-force-fields))
```

File: content/science/how-to-cite/_index.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+* Sage ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Zenodo DOIs](https://github.com/openforcefield/openff-forcefields#open-force-fields))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md]:
```suggestion
* Sage ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://github.com/openforcefield/openff-forcefields#open-force-fields))
```

File: content/science/how-to-cite/_index.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ 
+### Software
+
+* Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [latest version](https://zenodo.org/record/6310995#.YkdRtpPMLMI))
+
+* BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))
+
+* Evaluator ([Boothroyd et al. preprint](https://zenodo.org/record/5695016#.YkdTX5PMLMI) + [Zenodo DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```
Comment (j-wags) [inline content/science/how-to-cite/_index.md]: In addition to these changes, the link for the Boothroyd preprint is wrong below

```suggestion
* OpenFF Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [Specific version DOI](https://zenodo.org/record/6310995#.YkdRtpPMLMI))

* OpenFF BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))

* OpenFF Evaluator ([Boothroyd et al. preprint](https://zenodo.org/record/5695016#.YkdTX5PMLMI) + [Specific version DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```

**Review by j-wags — CHANGES_REQUESTED** [review]
Looks pretty good -- I updated the wording in a few places. Also, there's one link to a "Boothroyd et al." preprint that seems to be misdirected. I'll approve once that's linked correctly :-)

File: content/science/how-to-cite/_index.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+* Sage ([Qiu et al., 2021](https://pubs.acs.org/doi/10.1021/acs.jctc.1c00571) + [Specific version DOI](https://github.com/openforcefield/openff-forcefields#open-force-fields))
+ 
+### Software
+
+* OpenFF Toolkit ([Mobley et al., 2018](https://pubs.acs.org/doi/10.1021/acs.jctc.8b00640) + [Specific version DOI](https://zenodo.org/record/6310995#.YkdRtpPMLMI))
+
+* OpenFF BespokeFit ([Zenodo DOI](https://zenodo.org/record/5172174#.YkdU1JPMLMI))
+
+* OpenFF Evaluator ([Boothroyd et al. preprint](https://zenodo.org/record/5695016#.YkdTX5PMLMI) + [Specific version DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```
Comment (diegoonolasco) [inline content/science/how-to-cite/_index.md]:
```suggestion
* OpenFF Evaluator ([Boothroyd et al. preprint](https://chemrxiv.org/engage/chemrxiv/article-details/610ad0ed45805d722f80e4de) + [Specific version DOI](https://zenodo.org/record/5695016#.YkdTX5PMLMI))
```

**Review by j-wags — APPROVED** [review]
Looks great. Thanks @diegoonolasco!

**Pull request merged**